usaspending_api/disaster/v2/views/agency/loans.py

```diff
@@ -146,7 +147,7 @@ def _build_json_result(self, bucket: dict):
             "code": info["code"],
             "description": info["name"],
             # the count of distinct subtier agencies contributing to the totals
-            "count": int(bucket.get("doc_count", 0)),
+            "award_count": int(bucket.get("doc_count", 0)),
```

Comment:
I've reconfirmed with Seth. Because this is searching against the award index, the `doc_count` will a count of rows in that index, so this was always counting awards instead of agencies.